lib/services/ibm_devops.rb

```diff
@@ -1,8 +1,9 @@
 class Service::IBMDevOpsServices < Service::HttpPost
-  string :ibm_id
+  title 'IBM Bluemix DevOps Services'
+  string :ibm_login
```

Comment:
@kdaigle Thanks for the feedback.  Just to confirm then that this change will break for any users that have already configured the service?  So the service configurations themselves will not update when this change is rolled out?  If so then we will need to fall back as you suggested.